cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ServiceTask.java

```diff
@@ -82,6 +82,11 @@ public ServiceTask() {
         return this.processConfiguration.getJvmOptions();
     }
 
+    @Override
+    public Collection<String> getProcessParameters() {
+        return processConfiguration.getProcessParameters();
```

Comment:
this.

Reply:
Documentation? The JvmOptions also doesn't have any

Reply:
The same as above